database/index.js

```diff
@@ -78,4 +78,4 @@ const getImageInfo= function(callback) {
 
```

Comment:
also, on line 68, what did you mean by 'where rooms.id = 49' ?
i'm also confused a little on the structure of lines 53 to 74, but that is probably just because my arse hurts and my eyes are numb from staring at the monitor (and my lack of knowledge) so nothing makes sense to me - 
sorry for the questions, i'm not as knowledgeable in this as i would like to be....

i'm going to submit my review with what i've got and then commute home...on the train i'll take a look again and see if i can verbalize what my brain is trying to tell me.